app/repository/DisposeLiabilityReturnMongoRepository.scala

```diff
@@ -101,19 +102,19 @@ class DisposeLiabilityReturnReactiveMongoRepository(mongo: () => DB, val metrics
   }
 
   def deleteExpired60DayLiabilityReturns(batchSize: Int): Future[Int] = {
-    val query = BSONDocument("timeStamp" -> Json.obj("$lte" -> DateTime.now(DateTimeZone.UTC).withHourOfDay(0).minusDays(61).toString()))
+    val dayThreshold = 61
+    val jodaDateTimeThreshold = DateTime.now(DateTimeZone.UTC).withHourOfDay(0).minusDays(dayThreshold)
 
-    val logOnError = Cursor.ContOnError[Seq[DisposeLiabilityReturn]]((_, ex) =>
```

Comment:
this error logging could be carried over with call to .recover on the Future returned by query